geostacks/geostacks.py

```diff
@@ -11,78 +11,43 @@
 from datetime import datetime
 
 
-class SpatialIndex:
+class SpatialIndexL3:
```

Comment:
What does `L3` mean?

Reply:
Level 3 data -- i.e., gridded data scenes. The thought is that the indexing scheme will be similar (static footprint definitions in a data structure) for things like gridded MODIS and NCEP data, but different enough for a new class definition for swath data like non-gridded MODIS, or similar

Reply:
I thought we are working with Landsat Level 1 data but I get what you mean!